test/test_api.py

```diff
@@ -21,13 +20,9 @@ class TestTinEyeAPIRequest(unittest.TestCase):
     def setUp(self):
         self.api = TinEyeAPIRequest(
             api_url="https://api.tineye.com/rest/",
-            public_key="LCkn,2K7osVwkX95K4Oy",
-            private_key="6mm60lsCNIB,FwOWjJqA80QZHh9BMwc-ber4u=t^",
+            api_key="LCkn,2K7osVwkX95K4Oy",
```

Comment:
`            api_key="6mm60lsCNIB,FwOWjJqA80QZHh9BMwc-ber4u=t^",`
private key is now the only key